src/crypto/index.js

```diff
@@ -65,6 +65,9 @@ export const verificationMethods = {
     SAS: SAS.NAME,
 };
 
+const VERIFICATION_REQUEST_TIMEOUT = 5 * 60 * 1000; //5m
+const VERIFICATION_REQUEST_MARGIN = 3 * 1000; //3s
```

Comment:
Could use some docs here on what this is - I'm assuming the fudge factor I was talking about so we don't flash up requests and then hide them right after.